dhis2/src/test/java/org/motechproject/dhis2/event/EventHandlerTest.java

```diff
@@ -51,6 +57,8 @@
     @Mock
     private TrackedEntityInstanceMappingService trackedEntityInstanceMappingService;
     @Mock
+    private DataElementService dataElementService;
+    @Mock
     private DhisWebService dhisWebservice;
     private DhisStatusResponse response;
     private EventHandler handler;
```

Comment:
Just construct the instance here with the @InjectMocks annotation, and the need for constructors will go away.